contrib/endpoints/src/api_manager/context/service_context.h

```diff
@@ -108,6 +131,12 @@ class ServiceContext {
 
   // The service control object.
   std::unique_ptr<service_control::Interface> service_control_;
+
```

Comment:
not need to track using instances for a service config.  My current thinking is:  we can just leave the service in the memory even nobody is using it,   it will not take too much memory since client cache will be empty after idle for a white, and all items should be removed on expiration.